docs/index.qmd

```diff
@@ -24,17 +24,17 @@ pip install https://github.com/rstudio/py-shinyswatch/tarball/main
 
 ## Usage
 
-To use a theme, call the theme function and add it to your App's UI definition.
+To use a theme, add a `shinyswatch.theme` to your App's UI definition.
 
 ```python
 # Minty theme
-shinyswatch.theme.minty()
+shinyswatch.theme.minty
 
 # Sketchy theme
-shinyswatch.theme.sketchy()
+shinyswatch.theme.sketchy
 
 # Superhero theme
-shinyswatch.theme.superhero()
+shinyswatch.theme.superhero
```

Comment:
Please apply this change to all apps in `./examples` and `README.md`

Reply:
And https://github.com/posit-dev/py-shinyswatch/blob/4ef02f6387749a993024c2fad32133fc46607218/docs/_renderer.py#L32-L34